src/System.Buffers/src/Resources/Strings.resx

```diff
@@ -1,5 +1,64 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
+  <!-- 
```

Comment:
Why adding this garbage back?

Reply:
I hadn't realized we previously removed it (https://github.com/dotnet/corefx/pull/16284). I just added a new String resource in VS (used in an exception) and it auto-updated to add the boiler plate. I removed the unused string that was there previously.

Other resx file have it too:
https://github.com/dotnet/corefx/blob/master/src/System.Memory/src/Resources/Strings.resx
https://github.com/dotnet/corefx/blob/master/src/System.Text.Json/src/Resources/Strings.resx
https://github.com/dotnet/corefx/blob/master/src/System.IO.Pipelines/src/Resources/Strings.resx

@danmosemsft, should we be removing this comment? It looks like VS adds it back so not sure its worth it.

Reply:
My opinion is we shouldn't be fighting VS on it and shouldn't have removed them in the first place, since any modifications via the GUI end up adding it back.  Agreeing with @ericstj, I'm fine putting it back as part of this change as long as we leave it that way.  If VS ever allows us to configure the behavior, then we could set that configuration and do a one-time sweep to remove them from everywhere.

Reply:
I opened a bug to remove this 17 years ago. Until they do, there is no point fighting it.